docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -272,8 +282,8 @@ kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
 ```
 {% endcapture %}
 
-{% assign tab_names = "Choose one...,Calico,Canal,Flannel,Romana,Weave Net" | split: ',' | compact %}
-{% assign tab_contents = site.emptyArray | push: choose | push: calico | push: canal | push: flannel | push: romana | push: weave_net %}
+{% assign tab_names = "Choose one...,Calico,Canal,Flannel,Kube-router,Romana,Weave Net" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: choose | push: calico | push: canal | push: flannel | push: kube-router | push: romana | push: weave_net %}
```

Comment:
kube-router vs Kube-router (consistent usage)

Reply:
@radhikapc  I have made changes as suggested, except for one use of lower case kube-route, which is not part of displayed text, but its the template tabs internal name.

Reply:
Do you mean `kube-router` is used internally, whereas while referring to the service we use `Kube-router`?